pallets/gear-program/src/migration.rs

```diff
@@ -19,7 +19,10 @@
 use crate::{Config, Pallet, Weight};
 
 /// Wrapper for all migrations of this pallet, based on `StorageVersion`.
-pub fn migrate<T: Config>() -> Weight {
+pub fn migrate<T: Config>() -> Weight
+where
+    <T as frame_system::Config>::AccountId: common::Origin,
```

Comment:
import origin upper
...
```suggestion
    <T as Config>::AccountId: Origin,
```